### ISSUE TITLE
Got wifi unsupport

### ISSUE THREAD

**Issue by hamabao413** [dj1ch/minigotchi-ESP32#41]
**Describe the bug**
I Got wifi : unsupport fram type : 0c0
I Got wifi : unsupport fram type : 0a0
and 
Screen show Both packets failed to send

Dont know what I did err

**Comment by dj1ch** [comment]
> **Describe the bug** I Got wifi : unsupport fram type : 0c0 I Got wifi : unsupport fram type : 0a0 and Screen show Both packets failed to send
> 
> Dont know what I did err

esp-idf will block deauthentication frames by default. in this case you will need to apply the fixes stated in the install guide in [this section](https://github.com/dj1ch/minigotchi-ESP32/blob/main/INSTALL.md#step-2-building-and-flashing)

**Comment by dj1ch** [comment]
let me know if this fix works, I have applied this fix to my ESP32 already so it should.

**Comment by hamabao413** [comment]
> let me know if this fix works, I have applied this fix to my ESP32 already so it should.

I'm not sure if the format is incorrect when I editing platform.txt. Is there any demonstration?
I did:    build.extra_flags.esp32 -w
            build.extra_flags.esp32= -w
 both r err 

Thank you for help

**Comment by dj1ch** [comment]
> I'm not sure the format is incorrect when I editing platform.txt. Is there any demonstration?
> I did:
> build.extra_flags.esp32=-w -DARDUINO_USB_CDC_ON_BOOT=0
> compiler.c.elf.libs.esp32=-zmuldefs -lesp_ringbuf -lefuse -lesp_ipc -ldriver -lesp_pm -lmbedtls -lapp_update -lbootloader_support -lspi_flash -

As of now, we don't have any video guides. But from the look of it this is correct if that was the only modification you made. You're supposed to use esp32 version 2.0.10 not 2.0.11 because this can lead to unknown issues, which could be the cause of this problem. The fix is meant to be applied to that version only.

**Comment by ATOMNFT** [comment]
> > let me know if this fix works, I have applied this fix to my ESP32 already so it should.
> 
> I'm not sure the format is incorrect when I editing platform.txt. Is there any demonstration? I did:
> 
> build.extra_flags.esp32=-w -DARDUINO_USB_CDC_ON_BOOT=0
> 
> compiler.c.elf.libs.esp32=-zmuldefs -lesp_ringbuf -lefuse -lesp_ipc -ldriver -lesp_pm -lmbedtls -lapp_update -lbootloader_support -lspi_flash -
> 
> Same, I Got wifi : unsupport fram type : 0c0 　　　I Got wifi : unsupport fram type : 0a0
> 
> is that because i am using ESP32 version 2.0.11 not 2.0.10?
> 
> Also in INSTALL.md the II For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings III For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings are the same step?
> 
> **ESP8266 works for me and having fun**
> 
> Thank you for help and your great work!!

Yes, go to version 2.0.10
It is mentioned in the marauder tutorial (https://github.com/justcallmekoko/ESP32Marauder/wiki/arduino-ide-setup) that other versions will not work.

**Comment by dj1ch** [comment]
> > > let me know if this fix works, I have applied this fix to my ESP32 already so it should.
> > 
> > 
> > I'm not sure the format is incorrect when I editing platform.txt. Is there any demonstration? I did:
> > build.extra_flags.esp32=-w -DARDUINO_USB_CDC_ON_BOOT=0
> > compiler.c.elf.libs.esp32=-zmuldefs -lesp_ringbuf -lefuse -lesp_ipc -ldriver -lesp_pm -lmbedtls -lapp_update -lbootloader_support -lspi_flash -
> > Same, I Got wifi : unsupport fram type : 0c0 　　　I Got wifi : unsupport fram type : 0a0
> > is that because i am using ESP32 version 2.0.11 not 2.0.10?
> > Also in INSTALL.md the II For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings III For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings are the same step?
> > **ESP8266 works for me and having fun**
> > Thank you for help and your great work!!
> 
> Yes, go to version 2.0.10 It is mentioned in the marauder tutorial (https://github.com/justcallmekoko/ESP32Marauder/wiki/arduino-ide-setup) that other versions will not work.

I figured that was the issue anyway. I'll close this as soon as their ESP32 works correctly.

**Comment by hamabao413** [comment]
> > > > let me know if this fix works, I have applied this fix to my ESP32 already so it should.
> > > 
> > > 
> > > I'm not sure the format is incorrect when I editing platform.txt. Is there any demonstration? I did:
> > > build.extra_flags.esp32=-w -DARDUINO_USB_CDC_ON_BOOT=0
> > > compiler.c.elf.libs.esp32=-zmuldefs -lesp_ringbuf -lefuse -lesp_ipc -ldriver -lesp_pm -lmbedtls -lapp_update -lbootloader_support -lspi_flash -
> > > Same, I Got wifi : unsupport fram type : 0c0 　　　I Got wifi : unsupport fram type : 0a0
> > > is that because i am using ESP32 version 2.0.11 not 2.0.10?
> > > Also in INSTALL.md the II For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings III For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings are the same step?
> > > **ESP8266 works for me and having fun**
> > > Thank you for help and your great work!!
> > 
> > 
> > Yes, go to version 2.0.10 It is mentioned in the marauder tutorial (https://github.com/justcallmekoko/ESP32Marauder/wiki/arduino-ide-setup) that other versions will not work.
> 
> I figured that was the issue anyway. I'll close this as soon as their ESP32 works correctly.

OK thank you!!  I will try to install again.

**Comment by hamabao413** [comment]
> > > > > let me know if this fix works, I have applied this fix to my ESP32 already so it should.
> > > > 
> > > > 
> > > > I'm not sure the format is incorrect when I editing platform.txt. Is there any demonstration? I did:
> > > > build.extra_flags.esp32=-w -DARDUINO_USB_CDC_ON_BOOT=0
> > > > compiler.c.elf.libs.esp32=-zmuldefs -lesp_ringbuf -lefuse -lesp_ipc -ldriver -lesp_pm -lmbedtls -lapp_update -lbootloader_support -lspi_flash -
> > > > Same, I Got wifi : unsupport fram type : 0c0 　　　I Got wifi : unsupport fram type : 0a0
> > > > is that because i am using ESP32 version 2.0.11 not 2.0.10?
> > > > Also in INSTALL.md the II For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings III For compiler.c.elf.libs.esp32, compiler.c.elf.libs.esp32s2, compiler.c.elf.libs.esp32s3, compiler.c.elf.libs.esp32c3, add -zmuldefs to their compile settings are the same step?
> > > > **ESP8266 works for me and having fun**
> > > > Thank you for help and your great work!!
> > > 
> > > 
> > > Yes, go to version 2.0.10 It is mentioned in the marauder tutorial (https://github.com/justcallmekoko/ESP32Marauder/wiki/arduino-ide-setup) that other versions will not work.
> > 
> > 
> > I figured that was the issue anyway. I'll close this as soon as their ESP32 works correctly.
> 
> OK thank you!! I will try to install again.

i installed ESP32 by Espressif System 2.0.10 but is same just got wifi : unsupport fram type : 0c0

**Comment by dj1ch** [comment]
did you make the changes to platform.txt on `2.0.10`?

**Comment by hamabao413** [comment]
> did you make the changes to platform.txt on `2.0.10`?

Yes, I did and checked many times, Also tried Lilygo ESP32-S without display.

**Comment by dj1ch** [comment]
Are you compiling from the development branch or are you using a release? the latest ESP32 release has issues with deauthing(a couple months old)

**Comment by hamabao413** [comment]
I am using latest ESP32 release..... I thought that is NEW, I will try download files later. Thank you!

**Comment by hamabao413** [comment]
> > I am using latest ESP32 release..... I thought that is NEW, I will try download files later. Thank you!
> 
> no problem. at the moment I'm a little busy so most of my progress is in the development branch. once I can fix everything when I get the chance I can probably make a new release.

Confirmed that it works!!!!  Good Job ! I love it!

**Comment by dj1ch** [comment]
going to close the issue now that the problem is resolved.
